server/weaverbird/types.py

```diff
@@ -1,11 +1,12 @@
-from typing import Callable, List
+from typing import Any, Callable
 
 from pandas import DataFrame
 
 ColumnName = str
-Pipeline = List[dict]
+# Pipeline = List[dict]
 DomainRetriever = Callable[[str], DataFrame]
-PipelineExecutor = Callable[[List[dict]], DataFrame]
+PipelineExecutor = Callable[[Any], DataFrame]
```

Comment:
Maybe i should let PipelineExecutor#execute_pipeline take a list of dict, instead of a pipeline to avoid circular imports ?